docs/lineage/gravitino-spark-lineage.md

```diff
@@ -0,0 +1,109 @@
+| Iceberg catalog | `$GravitinoCatalogName.$schemaName.$tableName` | `iceberg_catalog.db.score` | 0.9.0         |
+| Paimon catalog  | `$GravitinoCatalogName.$schemaName.$tableName` | `paimon_catalog.db.detail` | 0.9.0         |
+| JDBC catalog    | `$GravitinoCatalogName.$schemaName.$tableName` | `jdbc_catalog.db.score`    | 0.9.0         |
+
+For datasets not managed by Gravitino, the dataset name is as follows:
+
+| Dataset Type | Dataset name                           | Example                               | Since Version |
+|--------------|----------------------------------------|---------------------------------------|---------------|
+| Hive         | `spark_catalog.$schemaName.$tableName` | `spark_catalog.db.table`              | 0.9.0         |
+| Iceberg      | `$catalogName.$schemaName.$tableName`  | `iceberg_catalog.db.table`            | 0.9.0         |
+| JDBC v2      | `$catalogName.$schemaName.$tableName`  | `jdbc_catalog.db.table`               | 0.9.0         |
+| JDBC v1      | `spark_catalog.$schemaName.$tableName` | `spark_catalog.postgres.public.table` | 0.9.0         |
+
+When accessing datasets by location (e.g., `SELECT * FROM parquet.$dataset_path`), the name is derived from the physical path:
+
+| Location Type  | Dataset name                                     | Example                               | Since Version |
+|----------------|--------------------------------------------------|---------------------------------------|---------------|
+| GVFS location  | `$GravitinoCatalogName.$schemaName.$filesetName` | `fileset_catalog.schema.fileset_a`    | 0.9.0         |
+| Other location | location path                                    | `hdfs://127.0.0.1:9000/tmp/a/student` | 0.9.0         |
+
+For GVFS location, the plugin add `fileset-location` facets which contains the location path.
+
+```json
+"fileset-location" :
+{
+"location":"/path/xx",
+"_producer":"https://github.com/datastrato/...",
+"_schemaURL":"https://raw.githubusercontent...."
+}
+```
+
+## How to use 
+
+1. Download Gravitino OpenLineage plugin jar and place it to the classpath of Spark.
+2. Add configuration to the Spark to enable lineage collection.
+
+Configuration example For Spark shell:
+
+```shell
+./bin/spark-sql -v \
+--jars /$path/openlineage-spark_2.12-$version.jar,/$path/gravitino-spark-connector-runtime-3.5_2.12-$version.jar \
+--conf spark.plugins="org.apache.gravitino.spark.connector.plugin.GravitinoSparkPlugin" \
+--conf spark.sql.gravitino.uri=http://localhost:8090 \
+--conf spark.sql.gravitino.metalake=$metalakeName \
+--conf spark.extraListeners=io.openlineage.spark.agent.OpenLineageSparkListener \
+--conf spark.openlineage.transport.type=http \
+--conf spark.openlineage.transport.url=http://localhost:8090 \
+--conf spark.openlineage.transport.endpoint=/api/lineage \
+--conf spark.openlineage.namespace=$metalakeName \
+--conf spark.openlineage.appName=$appName \
```

Comment:
Better change to `${}` for all the variables.

Reply:
updated